modules/ROOT/pages/upgrade.adoc

```diff
@@ -1,91 +1,11 @@
-= Upgrade Anypoint Platform Private Cloud
+= Upgrade Workflow
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-== Before You Begin
-Because Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrades require a working installation, all environment prerequisites should already be met. As long as you (or your infrastructure team) successfully performed all prerequisite tasks listed in xref:prereq-hardware.adoc[Hardware Prerequisites], xref:prereq-software.adoc[Software Prerequisites], and xref:prereq-network.adoc[Network Prerequisites] prior to the installation of Anypoint Platform PCE, your environment can be upgraded successfully.
+== MuleSoft Professional Services Required
```

Comment:
Should the following text mimic that used in the previous topic with the same title?

Reply:
ditto MuleSoft Professional Services, GLOBALLY. I will stop marking.